reporter/gen_report.js

```diff
@@ -32,11 +35,22 @@ Object.keys(originals).forEach(key => {
       escape(atob(originals[key]))));
 });
 
+var fileWriter = new FileWriter(false);
+
+fileWriter.writeFile = function writeFile(file, callback) {
+  if (file.length > 255) {
+    var excess = file.length - 255;
+    var baseName = path.basename(file, '.html')
```

Comment:
Yes, it also writes the base.css files from the report, png's etc (see screenshot below):

![screen shot 2016-09-14 at 17 36 32](https://cloud.githubusercontent.com/assets/1002056/18518822/d9746214-7aa1-11e6-9df6-addd10a23f42.png)